apps/api/src/middlewares/auth.ts

```
@@ -0,0 +1,118 @@
+  /* ──────────────────────────────────────────────────────────────── */
+  /* 1) Bearer plugin                                                */
+  /* ──────────────────────────────────────────────────────────────── */
+  .use(bearer())
+
+  /* ──────────────────────────────────────────────────────────────── */
+  /* 2) JWT plugin                                                   */
+  /* ──────────────────────────────────────────────────────────────── */
+  .use(
+    elysiaJwt({
+      name: "jwt",
+      secret: jwks as unknown as any, // Remote JWKS resolver (jose)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)